recipes/the-silver-searcher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "the-silver-searcher" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "ff7243863f22ed73eeab6f7a6d17cfff585a7eaa41d5ab3ae4f5d6db97701d5f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ggreer/{{ name | replace('-', '_') }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - automake
+    - pkg-config 
+    - pcre
+    - xz
+    - zlib
```

Comment:
These need to be pinned. Please see our [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/c09e5bafbc0b5e8eb7e93c06cf7403225bfa2016/scripts/pin_the_slow_way.py#L32-L73 ) for details.